added cleaned chunk 1

dg1223 opened the pull request:
This is the clean version of chunk 1 for যা দেখেছি যা বুঝেছি যা করেছি (#12 )

dg1223 committed:
added cleaned chunk 1

mir-abir-hossain commented on Task-1-Data-Extraction-Preprocess/books/যা_দেখেছি_যা_বুঝেছি_যা_করেছি/যা_দেখেছি_যা_বুঝেছি_যা_করেছি_1_clean.txt:
```diff
@@ -0,0 +1,2157 @@
+
+দেশব্যাপী আন্দোলনে অগ্রণীর ভূমিকায় ছিল জাগ্রত ছাত্র সমাজ ।
+
+'৭০ এর নির্বাচনের প্রেক্ষাপট। ট
```
The last extra 'ট' should be removed.

dg1223 replied:
fixed

mir-abir-hossain commented on Task-1-Data-Extraction-Preprocess/books/যা_দেখেছি_যা_বুঝেছি_যা_করেছি/যা_দেখেছি_যা_বুঝেছি_যা_করেছি_1_clean.txt:
```diff
@@ -0,0 +1,2157 @@
+নীতি । অবশিষ্ট বিষয়সমূহ স্টেট বা প্রদেশসমূহের হাতে থাকিবে ।
+
+(৩) পূর্ব এবং পশ্চিম পাকিস্তানের দুইটি পৃথক অথচ বিনিময় যোগ্য মুদ্রার প্রচলন
+করিতে হইবে৷ অথবা দুই অঞ্চলের জন্য একই কারেঙ্গি থাকিবে ৷ শাসনতন্ত্রে এমন
```
কারেঙ্গি  should be কারেন্সি  In avro it will be 'karensi' (There are same "engi" to "ensi" transformation needed in other places in the text)

dg1223 replied:
fixed

mir-abir-hossain commented on Task-1-Data-Extraction-Preprocess/books/যা_দেখেছি_যা_বুঝেছি_যা_করেছি/যা_দেখেছি_যা_বুঝেছি_যা_করেছি_1_clean.txt:
```diff
@@ -0,0 +1,2157 @@
+আগরতলা ষড়যন্ত্র মামলার ধারা বিবরণী খবরের কাগজে প্রকাশিত করতে থাকে
+আইয়ূব-মোনেম চক্র । তারা পূর্ব পাকিস্তানের বাঙ্গালী মুসলমানদের ভারত বিরোধী
+মনোভাবকে তাদের স্বার্থে কাজে লাগাবার উদ্দেশ্যেই এ সিদ্ধান্ত গ্রহণ করেছিলেন ৷
+পাকিস্তানের অখস্ততার বিরুদ্ধে ষড়যন্ত্রে লিপ্ত তখন তার বিরুদ্ধে বিক্ষোভে ফেটে
```
অখস্ততার should be অখন্ডতা. In avro it will be 'okhonDota'.

dg1223 replied:
fixed

mir-abir-hossain commented on Task-1-Data-Extraction-Preprocess/books/যা_দেখেছি_যা_বুঝেছি_যা_করেছি/যা_দেখেছি_যা_বুঝেছি_যা_করেছি_1_clean.txt:
```diff
@@ -0,0 +1,2157 @@
+মনোভাবকে তাদের স্বার্থে কাজে লাগাবার উদ্দেশ্যেই এ সিদ্ধান্ত গ্রহণ করেছিলেন ৷
+পাকিস্তানের অখস্ততার বিরুদ্ধে ষড়যন্ত্রে লিপ্ত তখন তার বিরুদ্ধে বিক্ষোভে ফেটে
+পড়বে। কিন্তু পরিণাম হল সম্পূর্ণ উল্টোটাই। প্রকাশিত ধারা বিবরণীর মধ্যেই
+পরিস্ষুটিত হয়ে উঠল আইয়ুব-মোনেম চক্রের ঘৃণ্য চক্রান্ত। এই ষড়যন্ত্রমূলক
```
পরিসষুটিত  should be পরিস্ফুটিত. In avro it will be 'porisfuTito'.

dg1223 replied:
fixed

mir-abir-hossain commented on Task-1-Data-Extraction-Preprocess/books/যা_দেখেছি_যা_বুঝেছি_যা_করেছি/যা_দেখেছি_যা_বুঝেছি_যা_করেছি_1_clean.txt:
```diff
@@ -0,0 +1,2157 @@
+ক্ষমতা দখলের নীল নকশা নতুন করে তৈরি করতে গোপন বৈঠকে মিলিত হতে
+থাকেন; অপরদিকে জেনারেল পীরজাদার মাধ্যমে ভুষ্টোকে উৎসাহিত করেন আইয়ূব
+বিরোধী গণআন্দোলন জোরদার করে তোলার জন্য । জেনারেলদের পরোক্ষ
+আশীরাদপুষ্ট হয়েই জনাব ভুষ্টো সরাসরিভাবে রাজনৈতিক অঙ্গনে আত্মপ্রকাশ করেন
```
আশীরাদপুষ্ট should be আশীর্বাদপুষ্ট. in avro it will be 'ashirrbadopuShTo'

dg1223 replied:
fixed

mir-abir-hossain reviewed (changes requested):
Please check the changes. See if they are necessary, if so then please change it, otherwise send pr

dg1223 committed:
made further corrections based on review

dg1223 commented:
> Please check the changes. See if they are necessary, if so then please change it, otherwise send pr

Fixed all issues that you listed. Found more issues and fixed them as well.

I have uploaded a [complete list of corrections](https://github.com/mir-abir-hossain/real-history-of-Bangladesh/wiki/Common-Data-Cleanup-Issues) to our Wiki.
